pages/print.js

```diff
@@ -91,6 +91,8 @@ const theme = createMuiTheme({
 
 export class Print extends Component {
   componentDidMount() {
+    const t = this.props.t;
+    document.title = t("titles.print");
```

Comment:
Could combine to one line `document.title = this.props.t("titles.print");`

Reply:
Oh, didn't that!